core/src/lib.rs

```diff
@@ -36,3 +110,21 @@ pub use graph::{Graph, Value};
 
 pub mod compile;
 pub use compile::{Compile, Compiler};
+
+#[cfg(test)]
+pub mod tests {
```

Comment:
There's no tests in this module!? If that's intentional, it should really be documented.